nav2_costmap_2d/include/nav2_costmap_2d/inflation_layer.h

```diff
@@ -80,9 +78,9 @@ class InflationLayer : public Layer
   virtual ~InflationLayer()
   {
     deleteKernels();
-    // TODO(bpwilcox): Resolve dynamic reconfigure dependencies
-    // if (dsrv_)
-    //    delete dsrv_;
+    if (dynamic_param_client_) {
```

Comment:
Could use a std::unique_ptr or std::shared_ptr to avoid having to explicitly delete this item.